libs/spark/src/Unstable_AvatarButton/Unstable_AvatarButton.stories.tsx

```diff
@@ -77,6 +86,10 @@ ChildrenInitialsSTP.args = { children: '(Initials)' };
 ChildrenInitialsSTP.decorators = [sparkThemeProvider];
 ChildrenInitialsSTP.storyName = 'children=(Initials) (STP)';
 
+export const ChildrenInitialsMedium: Story = Template.bind({});
+ChildrenInitialsMedium.args = { children: '(Initials)', size: 'medium' };
+ChildrenInitialsMedium.storyName = 'children=(Initials) (medium)';
```

Comment:
```suggestion
export const ChildrenInitialsSizeMedium: Story = Template.bind({});
ChildrenInitialsSizeMedium.args = { children: '(Initials)', size: 'medium' };
ChildrenInitialsSizeMedium.storyName = 'children=(Initials) size=medium';
```

Reply:
Will this not work with size: 'medium' format here? Or is this more following convention?

Reply:
this is following convention